openassessment/xblock/message_mixin.py

```diff
@@ -42,51 +42,55 @@ def render_message(self, data, suffix=''):
 
         is_closed = deadline_info.get('general').get('is_closed')
 
-        # Finds the status_information which describes the closed status of the current step (defaults to submission)
-        status_info = deadline_info.get(status, deadline_info.get("submission"))
-        status_is_closed = status_info.get('is_closed')
-
-        # Default context is empty
-        context = {}
-
-        # Default path leads to an "instruction-unavailable" block
-        path = 'openassessmentblock/message/oa_message_unavailable.html'
+        # Finds the deadline info of the current step (defaults to submission)
+        current_status_deadline = deadline_info.get(status, deadline_info.get("submission"))
```

Comment:
Nit: I found this name confusing, as it doesn't represent a deadline. Isn't it more like ``current_deadline_status``?